server/migrations/versions/1ca51eb40533_refactor_models_singular_names_.py

```
@@ -0,0 +1,142 @@
+
+# revision identifiers, used by Alembic.
+revision = '1ca51eb40533'
+down_revision = '3a5063766c1b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('mpesa_transaction',
+    sa.Column('id', sa.Integer(), nullable=False),
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)